forge/src/main/java/dev/schhmarrn/lighty/forge/LightyForge.java

```diff
@@ -0,0 +1,55 @@
+package dev.schhmarrn.lighty.forge;
+
+import dev.schmarrn.lighty.Lighty;
+import dev.schmarrn.lighty.event.Compute;
+import dev.schmarrn.lighty.event.KeyBind;
+import net.minecraft.client.Minecraft;
+import net.minecraftforge.api.distmarker.Dist;
+import net.minecraftforge.client.event.RenderLevelStageEvent;
+import net.minecraftforge.event.TickEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.event.lifecycle.FMLLoadCompleteEvent;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+@Mod(Lighty.MOD_ID)
+public class LightyForge {
+
+    public LightyForge() {
+        Lighty.init();
+    }
+
+    @Mod.EventBusSubscriber(bus = Mod.EventBusSubscriber.Bus.MOD, value = Dist.CLIENT)
```

Comment:
Please put a comment with a link to documentation where someone not experienced in forge things (me) can look at the forge magic (applies to all other forge magic thingies as well)

Reply:
The documentation is already present in the javadoc of the anotation

Reply:
But for that I'd need to open up the IDE, which I didn't for now